WTWSMS/history/characters/parthian.txt

```diff
@@ -510,7 +510,6 @@
 	religion="zoroastrian"
 	culture="parthian"
 	father=214055
-	trait=mazdeist
```

Comment:
Why did you remove this?

Reply:
When first time i worked on the Arsacids, i wanted to add zoroastrian school trait to them. But then, because i didn't found anything that clearly says about belonging of the Arsacids to any school, i changed my mind and removed traits that i previously added. Thereat i forgot to removed school traits from these characters, so i done it now. I think we could made them Zurvanists, as an oppose to the Mazdeist Sassanians.

Reply:
Ok, the player and AI gets a choice when starting their game.